pkg/rotator/rotator_test.go

```diff
@@ -1,10 +1,19 @@
 package rotator
 
 import (
+	"context"
+	"fmt"
 	"testing"
 	"time"
 
+	"github.com/onsi/gomega"
```

Comment:
Any reason you didn't dot-import here (e.g. `. "github.com/onsi/gomega"`)? I kind of prefer saying `HaveOccurred()` and not `gomega.HaveOccurred()`, and I thought it was standard, but I don't feel too strongly about it.

Reply:
I tend to avoid dot imports, as per the reasoning given in [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#import-dot):

> Except for this one case, do not use import . in your programs. It makes the programs much harder to read because it is unclear whether a name like Quux is a top-level identifier in the current package or in an imported package.

Reply:
Fair enough. I've seen that suggestion fairly widely ignored in test code only but I'm good either way.